### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by konkarong** [sampotts/plyr#2349] [Q&A]
Hi!
Sorry for dumb question... But how do I update this? I don't have the plugin, just the script.

**Comment by konkarong** [comment]
Can someone please help me?

**Comment by sampotts** [comment]
Hey,

How are you including the script? Using a `<script>` element? If so, you need to update the path in there to:

```
<script src="https://cdn.plyr.io/3.6.9/plyr.js"></script>
```

...or if you want polyfills (e.g. you're supporting older browsers)...

```
<script src="https://cdn.plyr.io/3.6.9/plyr.polyfilled.js"></script>
```

If you're including the CSS in the same way, then...

```
<link rel="stylesheet" href="https://cdn.plyr.io/3.6.9/plyr.css" />
```

**Reply by sampotts** [reply]
It's worth noting the version in those URLs will change in future (e.g. `3.6.9` to `3.7.0`) and that's the only part that needs updating.

**Reply by konkarong** [reply]
Hi and thank you for your answer!
The problem is that I can't find the script somewhere, but now I'm starting to wonder if our videos not working because of this (but I tried to change the url to https://vimeo.com/api/oembed.json?url=https%3A//vimeo.com/[id] but it made no difference):
https://github.com/sampotts/plyr/issues/1181#issuecomment-695857688
